rasa/nlu/test.py

```diff
@@ -834,8 +905,9 @@ def evaluate_entities(
     entity_results: List[EntityEvaluationResult],
     extractors: Set[Text],
     output_directory: Optional[Text],
-    successes: bool = False,
-    errors: bool = False,
+    successes: bool,
+    errors: bool,
```

Comment:
Given that we're removing the default value here, could this have an impact on external users of this function? Is this a function that users could usually call directly via Python?

Reply:
I doubt it. Most likely you would use `test_nlu()` in `rasa.test`. I wanted to make the signature consistent with `evaluate_intents` and `evaluate_responses`. Also, this is going to be released with 2.0. So, changing the defaults of a function that should most likely not be used by any users directly is fine (I hope).